README.md

```diff
@@ -0,0 +1,219 @@
+# binge - version-less JavaScript package management for monorepos
+Binge is a version-less JavaScript package management tool for monolithic repositories (*monorepos*).
+
+It helps you to:
+* encourage modularization for a clearer separation of concerns,
+* enforce dependency version consistency across different local packages,
+* improve the developer experience and reduces the build file size by ensuring there is only one version of a dependency across all packages in the repository.
+
+[//]: # (To learn more about *why exactly* you should consider using binge, read our [announcement blog post]https://tech.signavio.com/2017/package-management-binge.)
+
+## Getting started with binge
+Install binge by running ``yarn global add binge``.
+
+**Note**: binge requires [Yarn](https://yarnpkg.com/lang/en/) and doesn't support npm.
+
+### Repository structure
+Structure your repository as follows:
+
+```
+…
+- client
+    - package1
+        - …
+        - src
+        - .babelrc
+        - .npmignore
+        - package.json
+        - yarn.lock
+    - …
+    - packageN
+        - …
+    - app
+        - src
+        - package.json
+        - …    
+…
+```
+
+Your client folder needs to contain a root-level folder for each local package, plus one additional folder that contains the global entry point to (the *root* of) your app.
+In our example (see: [./examples/watch-app](./examples/watch-app)), we name this folder ``app``.
```

Comment:
This is not mandatory. The structure is totally up to you. Just do not host any package inside another, even tho is it is supported by the tool. It is actually an upside concerning lerna, there is no structure for packages.

In the case of the signavio monorepo we have the dummy folder (client/_) only to coordenate stuff like the harmony. Otherwise harmony wouldn't cover between apps, like: portal could use react 16 and explorer react 15, exmaple